multichat/chat/settings.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+
+NOTIFY_USERS_ON_ENTER_OR_LEAVE_ROOMS = getattr(settings, 'NOTIFY_USERS_ON_ENTER_OR_LEAVE_ROOMS', True)
+
+MSG_TYPE_MESSAGE = 0  # For standard messages
+MSG_TYPE_WARNING = 1  # For yellow messages
```

Comment:
This and ALERT don't have JS implementations; I'd rather they were either fully implemented or removed.